static/js/biomedical/disease/page.tsx

```diff
@@ -19,78 +19,92 @@
  */
 
 import axios from "axios";
+import _ from "lodash";
 import React from "react";
 
 import { MapTile } from "../../components/tiles/map_tile";
 import { USA_NAMED_TYPED_PLACE } from "../../shared/constants";
-import { GraphNodes } from "../../shared/types";
 import { getEntityLink } from "../bio_charts_utils";
 import {
   drawDiseaseGeneAssocChart,
   drawDiseaseSymptomAssociationChart,
 } from "./chart";
 import {
+  doesDiseasePrevalenceIDexist,
   getCompoundDiseaseContraindication,
   getCompoundDiseaseTreatment,
   getDiseaseCommonName,
   getDiseaseGeneAssociation,
   getDiseaseSymptomAssociation,
 } from "./data_processing_utils";
 import { DrugTreatmentTable } from "./drug_table";
+import {
+  CompoundDiseaseContraindicationData,
+  CompoundDiseaseTreatmentData,
+  DiseaseGeneAssociationData,
+  DiseaseSymptomAssociationData,
+} from "./types";
+
+const DISEASE_TREATMENT_COLUMNS = [
+  { id: "node", name: "Parent Node" },
+  { id: "id", name: "Compound ID" },
+  { id: "name", name: "Compound Name" },
+  { id: "clinicalPhaseNumber", name: "FDA Clinical Phase" },
+];
+const DISEASE_CONTRAINDICATION_COLUMNS = [
+  { id: "node", name: "Parent Node" },
+  { id: "id", name: "Compound ID" },
+  { id: "name", name: "Compound Name" },
+  { id: "drugSource", name: "Drug Source" },
+];
 
 export interface PagePropType {
   dcid: string;
   nodeName: string;
 }
 
+// fields for each of the charts, ex:
 export interface PageStateType {
-  data: GraphNodes;
+  diseaseGeneAssociation: DiseaseGeneAssociationData[];
+  diseaseSymptomAssociation: DiseaseSymptomAssociationData[];
+  chemicalCompoundDiseaseTreatment: CompoundDiseaseTreatmentData[];
+  chemicalCompoundDiseaseContraindication: CompoundDiseaseContraindicationData[];
+  diseaseCommonName: string;
+  diseasePrevalenceIDexists: boolean;
+  dataFetched: boolean;
 }
 
 export class Page extends React.Component<PagePropType, PageStateType> {
   constructor(props: PagePropType) {
     super(props);
-    this.state = { data: null };
+    this.state = {
+      diseaseGeneAssociation: null,
+      diseaseSymptomAssociation: null,
+      chemicalCompoundDiseaseTreatment: null,
+      chemicalCompoundDiseaseContraindication: null,
+      diseaseCommonName: null,
+      diseasePrevalenceIDexists: null,
+      dataFetched: null,
```

Comment:
nit: can just set this to false initially?

Reply:
Right! Set it to false!